builder/dockerfile/buildargs_test.go

```diff
@@ -37,7 +37,7 @@ func TestGetAllAllowed(t *testing.T) {
 		"ArgWithDefaultInDockerfile":          "fromdockerfile1",
 		"ArgNoDefaultInDockerfileFromOptions": "fromopt3",
 		"ArgFromMeta":                         "frommeta1",
-		"ArgFromMetaOverriden":                "fromdockerfile3",
```

Comment:
(I thought I pushed an update with this, I guess it'll have to wait until evening)